src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.tpl.html

```diff
@@ -11,7 +11,14 @@
         <button class="btn btn-xs btn-link" ng-click="$ctrl.openPaymentMethodFormModal(paymentMethod)" ng-if="paymentMethod['card-type']" translate>edit</button>
       </label>
     </div>
-
+    <div ng-if="$ctrl.selectedPaymentMethod['card-type']" class="row">
+      <div class="col-sm-4">
+        <form novalidate name="$ctrl.creditCardPaymentForm">
+          <credit-card-cvv ></credit-card-cvv>
```

Comment:
It doesn't seem like the cvv is getting stored in sessionStorage here like it does in other cases, which means it probably isn't getting sent to the server properly. This is due to the unsetting of the cvv in `selectPayment` but never re-setting it anywhere else (e.g. on hitting continue)

Reply:
bump